readme.md

```diff
@@ -20,4 +20,12 @@ This repository is licensed under the Apache 2.0 License, except for the [server
 
 ## Development
 
+To contribute to the project see https://www.mattermost.org/contribute-to-mattermost.
```

Comment:
That page seems quite generic. Someone who is looking into contributing to a plugin might not be interested in other parts of Mattermost. Maybe move this at the end of the section and make clear that it's about other Mattermost projects?